README.md

```diff
@@ -1,2 +1,17 @@
-# mv-08-htmlcss-framework
-Responsive 12 Columns CSS Grid Framework
+# Responsive 12 Columns Grid Framework
+
+A Bootstrap like 12 columns responsive framework.
```

Comment:
**[REQUIRED]** Please, make the readme file more descriptive, find some inspiration [here](https://github.com/othneildrew/Best-README-Template). A good readme file is essential to give a good impression, try to talk about what your framework does, how can be used, etc.